SD-K9/src/app/app-routing.module.ts

```diff
@@ -3,14 +3,6 @@ import { PreloadAllModules, RouterModule, Routes } from '@angular/router';
 
 const routes: Routes = [
   { path: '', redirectTo: 'home', pathMatch: 'full' },
-  {
-    path: 'bus',
-    loadChildren: () => import('src/app/components/bus/bus.module').then(m => m.BusPageModule)
-  },
-  {
-    path: 'settings',
-    loadChildren: () => import('src/app/pages/app-settings/app-settings.module').then(m => m.AppsettingsPageModule)
-  },
```

Comment:
Why did you remove the routing for bus.page and settings.page?

Reply:
The bus and settings are modal components, they do not need to be accessed by the routing and only by the modal button. This button is on the pull-up bar component. The code was giving me errors when it was there.